frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -489,8 +477,36 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                 setCompareFilter: (_, { compareFilter }) => compareFilter,
             },
         ],
+        productTab: [
+            ProductTab.ANALYTICS as ProductTab,
+            {
+                setProductTab: (_, { tab }) => tab,
+            },
+        ],
     }),
     selectors(({ actions, values }) => ({
+        breadcrumbs: [
```

Comment:
I'm still not sure how `breadcrumbs` work. Who's accessing these? This is like magic to me

Reply:
Whatever is your active scene, and what is that scene's logic and component, is recorded in `sceneLogic` (each scene exports its data with `export const scene: SceneExport = ...`). If the scene's logic contains a `breadcrumbs` selector, we use it to show breadcrumbs via `breadcrumbsLogic`

Reply:
> If the scene's logic contains a breadcrumbs selector

Ah ok, that's the confusing bit for me. So we're **dynamically** accessing the logic exported on `SceneExport` and then conditionally checking if they have a `breadcrumbs` selector? Sounds like typescript hell, but that makes sense. And then, if there's no selector, we simply use the `SceneExport` title as the crumb, right?